package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
Seems that I also need to put some comments here. The reason I removing rollup from package.jsin is because I can not rely on latest rollup version - with the latest rollup I have some strange issues while making [PKIjs](https://github.com/PeculiarVentures/pkijs-es6) projects/examples. Thus I need to use 0.34.7 instead of any rollup versions using by rollem.

Removing rollup dependency would give much more flexibility to rollem, I do believe so.